sig-provider/sig-provider-proto/build.rs

```diff
@@ -17,6 +17,14 @@ fn compile(
         .protoc_arg("grpc_api_configuration=proto/api_config_http.yaml,output_format=yaml,allow_merge=true,merge_file_name=sig-provider")
         .bytes(["."])
         .type_attribute(".", "#[actix_prost_macros::serde]");
+
+    for path in protos.iter() {
+        println!("cargo:rerun-if-changed={}", path.as_ref().display())
+    }
```

Comment:
Just interested, what these lines are for?

Reply:
by default cargo will always rebuild the crate with custom build.rs
this stops it rebuilding if this files didn't change from the last build